src/components/topRail/topRail.scss

```diff
@@ -0,0 +1,9 @@
+.top-rail-container {
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: {
+    left: 8px;
+    right: 8px;
+  };
```

Comment:
this is incorrect. should be `padding: 0 8xp 0 8px;` or `padding: 0 8px`

Reply:
It will be fixed